src/test/java/com/github/caldav4j/CalDAVCollectionTest.java

```diff
@@ -212,39 +198,37 @@ public void testGetCalendarByPath() throws Exception {
 	 */
 	@Test
 	public void testGetEventResources() throws Exception {
-		Date beginDate = ICalendarUtils.createDateTime(2006, 0, 1, null, true);
-		Date endDate = ICalendarUtils.createDateTime(2006, 0, 9, null, true);
-		Thread.sleep(1000);
```

Comment:
This sleep was introduced to properly ensure the event was consistently stored on the server (eg. distributed / eventually consistent server deployment).